google-cloud-clients/google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITStorageTest.java

```diff
@@ -361,17 +355,37 @@ public void testGetBucketEmptyFields() {
 
   @Test
   public void testGetBucketLifecycleRules() {
-    Bucket remoteBucket = storage.get(BUCKET, Storage.BucketGetOption.fields(BucketField.values()));
+    String lifecycleTestBucketName = RemoteStorageHelper.generateBucketName();
+    storage.create(
+            BucketInfo.newBuilder(lifecycleTestBucketName)
+                    .setLocation("us")
+                    .setLifecycleRules(
+                            ImmutableList.of(
+                                    new BucketInfo.LifecycleRule(
+                                            new BucketInfo.LifecycleRule.SetStorageClassLifecycleAction(
+                                                    StorageClass.COLDLINE),
+                                            new BucketInfo.LifecycleRule.LifecycleCondition.Builder()
+                                                    .setAge(1)
+                                                    .setNumberOfNewerVersions(3)
+                                                    .setIsLive(false)
+                                                    .setCreatedBefore(new DateTime(System.currentTimeMillis()))
+                                                    .setMatchesStorageClass(ImmutableList.of(StorageClass.COLDLINE))
+                                                    .build())))
+                    .build());
+    Bucket remoteBucket = storage.get(lifecycleTestBucketName, Storage.BucketGetOption.fields(BucketField.LIFECYCLE));
     BucketInfo.LifecycleRule lifecycleRule = remoteBucket.getLifecycleRules().get(0);
-
-    assertTrue(
-        lifecycleRule.getAction()
-            instanceof BucketInfo.LifecycleRule.SetStorageClassLifecycleAction);
-    assertEquals(3, (int) lifecycleRule.getCondition().getNumberOfNewerVersions());
-    assertNotNull(lifecycleRule.getCondition().getCreatedBefore());
-    assertFalse(lifecycleRule.getCondition().getIsLive());
-    assertEquals(1, (int) lifecycleRule.getCondition().getAge());
-    assertEquals(1, lifecycleRule.getCondition().getMatchesStorageClass().size());
+    try {
+      assertTrue(
+          lifecycleRule.getAction()
```

Comment:
This could also be reduced to:
`assertTrue(lifecycleRule.getAction().lifecycleAction().equals( BucketInfo.LifecycleRule.SetStorageClassLifecycleAction.TYPE));`